src/app/ideas/IdeaPost.js

```diff
@@ -14,12 +14,16 @@ const IdeaText = styled.p`
   font-size: 0.8em;
 `;
 
-export default function IdeaPost({ title, text, tags }) {
+export default function IdeaPost({ title, text, tags, handleTagClick }) {
   return (
     <IdeaContainer>
       <IdeaTitle>{title}</IdeaTitle>
       <IdeaText>{text}</IdeaText>
-      {tags && tags.map(tag => <Tag key={tag} tag={tag} />)}
+
+      {tags &&
+        tags.map(tag => (
+          <Tag key={tag} tag={tag} handleTagClick={handleTagClick} />
```

Comment:
maybe you can name your prop `tagName`. otherwise it might be a little confusing because your component has the same name as the key value as the prop as the prop value :)

Reply:
I'll think about it, I agree that it may be confusing, but I'd rather like to rethink the entire Naming Conventions and go over for improvements at a later stage.

Reply:
OK, I fixed that one for now...